examples/Widget Basics.ipynb

```diff
@@ -196,7 +196,7 @@
    "source": [
     "Widgets are represented in the back-end by a single object.  Each time a widget is displayed, a new representation of that same object is created in the front-end.  These representations are called views.\n",
     "\n",
-    "![Kernel & front-end diagram](images/WidgetModelView.png)"
```

Comment:
Can you summarize the main points of the PR in a comment?

On Thu, Apr 7, 2016 at 10:55 AM, Jonathan Frederic <notifications@github.com
> wrote:

> In examples/Widget Basics.ipynb
> <https://github.com/ipython/ipywidgets/pull/464#discussion_r58915868>:
>
> > @@ -196,7 +196,7 @@
> >     "source": [
> >      "Widgets are represented in the back-end by a single object.  Each time a widget is displayed, a new representation of that same object is created in the front-end.  These representations are called views.\n",
> >      "\n",
> > -    "![Kernel & front-end diagram](images/WidgetModelView.png)"
>
> wrong
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipywidgets/pull/464/files/01034f4d9f5da7684f2f7bd4da099cc6b6ad5234#r58915868>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com